extensions/basic/jruby-ext/src/main/java/com/purbon/BasicService.java

```diff
@@ -10,23 +10,40 @@
 import java.io.IOException;
 
 /**
- * This class is used when to run the instanciation and load
- * of all the related modules and classes defined here.
+ * This class is used to load all related modules in a lib (metaphor in the extensions mechanism).
+ * Everything defined in the load method would be instantiated when teh ruby code does the require
+ * poiting to this service. (Example require "com/purbon/basic")
  * Created by purbon on 24/08/15.
```

Comment:
It may be useful to lay out in this javadoc what the equivalent module definition in Ruby would be, e.g.

```ruby
module Foo
  class Bar
    # etc
  end
end

```